data-import/desktop-csv-import/desktop-csv-import.adoc

```diff
@@ -0,0 +1,420 @@
+--
+
+Just as we did with the last CSV file, let us look at the results of these conversions without importing the data.
+
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///desktop-csv-import/orders.csv' AS row
+RETURN toInteger(row.orderID), datetime(replace(row.orderDate,' ','T')), row.shipCountry
+LIMIT 5
+----
+
+image::{img}desktop_import_orders_convert.jpg[link="{img}desktop_import_orders_convert.jpg",role="popup-link"]
+
+There was one tricky thing with this CSV in the `orderDate` column.
+Neo4j's datetime format will accept many formats, but the time value must start with a `T`.
+The CSV file did not have the 'T' joining the date and time values but had a space between them instead.
+We used the `replace()` function to change the space to the character 'T' and get the string into the expected format.
+Then, we wrapped the `datetime()` function around that to convert the changed string to a datetime value.
+
+.Order-details.csv
+The values in the order-details.csv (from column names) are for productID, orderID, and quantity.
+Let us look at which ones will need to be converted.
+
+Our product ID is also from our products.csv file, where we converted that value to an integer.
+We will do the same here to ensure we match formats.
+The order ID field contains values from our orders.csv file, so we will match our previous conversion and translate this field to an integer, as well.
+The quantity field in this file is a numeric value.
+We can convert this to an integer with the `toInteger()` function we have been using.
+
+The results of these conversions are in the code below.
+Remember that we still are not loading any data yet.
+
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///desktop-csv-import/order-details.csv' AS row
+RETURN toInteger(row.productID), toInteger(row.orderID), toInteger(row.quantity)
+LIMIT 8
+----
+
+image::{img}desktop_import_details_convert.jpg[link="{img}desktop_import_details_convert.jpg",role="popup-link"]
+
+== Loading the data!
+
+Now that we have determined that the CSV file data looks ok, and we have verified how `LOAD CSV` sees the data and converted any non-string values, we are almost ready to create the data in our graph database!
+To do that, we use Cypher statements alongside the `LOAD CSV` commands we used above.
+The `LOAD CSV` will read the files, and the Cypher statements will create the data in our database.
+
+=== Graph data model
+
+An important step we need before writing Cypher statements, though, is to determine what the graph structure should look like once we import our file data.
+After all, importing the data in the existing table and column data will not provide the value we want to achieve from a graph.
+To utilize the graph database fully, we need a graph data model.
+
+We do not need to analyze all the various ways we could organize the products and orders in our files, so we have provided the model we will use for this exercise below.
```

Comment:
Depends on the intended use-case what kind of shape we want/need.

Reply:
What would be the use case for using a graph as a relational data store, though (or document or any model)? Or am I misunderstanding?